tests/unit/test_base_test.py

```diff
@@ -161,3 +446,120 @@ def test_exeption_in_assert_util(ucc_mock, exception):
     with patch("builtins.callable", side_effect=exception("important msg")):
         with pytest.raises(Exception, match="important msg"):
             ucc_mock.assert_util("left", "right")
+
+
+def test_setup_class_ucc_tester():
+    ucc_tester = UccTester()
+    ucc_tester.setup_class()
+    assert isinstance(ucc_tester.wait, selenium.webdriver.support.wait.WebDriverWait)
+    assert ucc_tester.wait._timeout == 20
+
+
+SESSION_KEY = "1234"
+
+
+@pytest.fixture()
+def mock_request():
+    mock = MagicMock()
+    mock.json = lambda: {"sessionKey": SESSION_KEY}
+    return mock
+
+
+defaults_rest_helper = {
+    "splunk_mgmt_url": "https://localhost:8089",
+    "username": "admin",
+    "password": "Chang3d!",
+}
+
+
+def test_rest_helper(mock_request):
+    with patch("requests.post", return_value=mock_request):
+        rest_helper = RestHelper(**defaults_rest_helper)
+    assert rest_helper.session_key == SESSION_KEY
+    assert rest_helper.splunk_mgmt_url == defaults_rest_helper.get("splunk_mgmt_url")
+    assert rest_helper.username == defaults_rest_helper.get("username")
+    assert rest_helper.password == defaults_rest_helper.get("password")
+
+
+@pytest.mark.parametrize(
+    "side_effect",
+    [
+        lambda: (_ for _ in ()).throw(Exception()),
```

Comment:
just Exception as side_effect does not work?
`mock_request.json = MagicMock(side_effect=Exception)`

Reply:
updated